styles/action.less

```diff
@@ -63,6 +63,22 @@ nav.action-group {
     border-bottom-right-radius: var(--ctrl-border-radius);
 }
 
+@keyframes action-loading-spinner {
+    from { transform: rotate(-0.25turn); }
+    to { transform: rotate(0.25turn); }
+}
+
+.action-container .loading img {
+    border-radius: 50%;
+    border: 2px dotted  #aaa;
+    border-top-color: #444;
+    border-bottom-color: #444;
+    animation-name: action-loading-spinner;
+    animation-duration: 1.5s;
+    animation-iteration-count: infinite;
+    animation-timing-function: ease-in-out;
+    animation-direction: alternate;
+}
```

Comment:
You can use the existing spinner that's used on the deleted-items page:

https://github.com/josh-berry/tab-stash/blob/12ae9149ad83e0306e32a2220406964d780886b7/src/deleted-items/bookmark.vue#L3

Given that pretty much any item-icon can be in a loading state at this point, you might want to consider adding support for a loading state in the item-icon component itself:

https://github.com/josh-berry/tab-stash/blob/master/src/components/item-icon.vue

Reply:
You know what? Nevermind, I've been playing with it a bit and I like your spinner better than mine. 🤣 But I think we should probably merge the spinner in deleted-items with this one.

Reply:
Sorry, one more minor thing--it doesn't work with the "default" tab icon, because the default icon isn't actually an `<img>`; it using the background-image trick to pick the right icon depending on the theme (light or dark).

Reply:
I mostly came up with that because I couldn't figure out how to superimpose a spinner over the icon without screwing up the layout, but I ended up liking it too!

I fixed the issue with the default tab icon.

I didn't mess with the `item-icon` component at this point because I'm afraid of breaking other stuff and I don't really have the time to look at it in depth right now. 

As for the color variables, do you mean adding some to the themes? Right now it has the half light/half dark border which stands out pretty well in both themes when I tested them. If you have a suggestion for existing variables to use for the dark/light parts of the border, I can certainly change it to use that. I looked through them and I didn't see anything that was both close to the existing colors and would be logical to use.

Reply:
> I didn't mess with the `item-icon` component at this point because I'm afraid of breaking other stuff and I don't really have the time to look at it in depth right now.

Fair enough. I don't know that there's actually much logic that could be pushed down anyway.

> As for the color variables, do you mean adding some to the themes? Right now it has the half light/half dark border which stands out pretty well in both themes when I tested them. If you have a suggestion for existing variables to use for the dark/light parts of the border, I can certainly change it to use that. I looked through them and I didn't see anything that was both close to the existing colors and would be logical to use.

Hm, if you want to keep the half-light/half-dark motif, I think my best suggestion would be `--page-fg` and `--disabled-fg` (used for page text and disabled text, respectively). Or you could just stick entirely with `--page-fg` (which would match the color of the default iconography and text). But I also agree the colors you picked work well in both modes.

Reply:
>  if you want to keep the half-light/half-dark motif, I think my best suggestion would be `--page-fg` and `--disabled-fg`

I do! I need my PRs to make some sort of deep statement on the duality of man.    
<sub>*… or maybe I just like how it looks more better a solid color.*</sub>

I changed to the variables you suggested and it looks okay to me in dark/light themes so I'm good with that. 

***

By the way, is it just me or is it impossible to see some favicons in the light theme? I went back and tested the version without my changes and that still seem to be the case. Example:

Dark|Light
-|-
![image](https://user-images.githubusercontent.com/44031344/158018087-fdb38181-819d-4307-8c3d-80ff003a8132.png)|![image](https://user-images.githubusercontent.com/44031344/158018101-1c5f9163-645c-4bbf-bc44-41084b451f95.png)

Reply:
> > if you want to keep the half-light/half-dark motif, I think my best suggestion would be `--page-fg` and `--disabled-fg`
> 
> I do! I need my PRs to make some sort of deep statement on the duality of man. _… or maybe I just like how it looks more better a solid color._

And here I thought you were a back-end person. 🤣

> By the way, is it just me or is it impossible to see some favicons in the light theme? I went back and tested the version without my changes and that still seem to be the case. ...

I think this is a function of some sites detecting the dark/light theme of your system and changing their favicon to match... except if you switch Tab Stash to the opposite theme, then it *doesn't* match. I have the same problem in reverse--e.g. GitHub gives me a dark icon since my system uses a light theme, and so when I switch Tab Stash to use the dark theme, I get black on black.

I don't have this problem if all my themes are matching (system theme == Tab Stash theme).

Reply:
> And here I thought you were a back-end person. :rofl:

Haha, I am! I'm just a back-end person with strong opinions about things I don't know much about!

> I think this is a function of some sites detecting the dark/light theme of your system and changing their favicon to match...

Oh, that makes perfect sense. Thanks!

Reply:
> > And here I thought you were a back-end person. 🤣
> 
> Haha, I am! I'm just a back-end person with strong opinions about things I don't know much about!

That makes two of us! 😂